### DISCUSSION TITLE
모바일 페이지 이동 번호가 안 뜨는 영상

### DISCUSSION THREAD

**Discussion by Jaeppetto** [pronist/hELLO#196] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/95916813/218507239-4c53b297-6e41-49f8-9853-59041c9201b1.png)

안녕하세요. 이쁜 스킨 만들어 주셔서 감사드립니다. 다름이 아니라 모바일 화면에서 최하단 페이지 번호가 < 1 > 밖에 안 떠서 해결법을 여쭙고자 discussion에 글 남깁니다. 

메인이나 다른 카테고리 불문하고 문제가 발생합니다.

**Comment by pronist** [comment, marked as answer]
@w00ngja 안녕하세요. 

모바일에서 페이징에 그렇게 나오는 것은 버그가 아니라 기능입니다. 즉, 제가 의도적으로 그렇게 한 겁니다. 따라서 **문제**가 아니므로 **해결**하지 않아도 됩니다.

저렇게 나오는 것이 문제라고 생각되신다면 **페이징**에 관련된 CSS 를 수정하시면 되겠습니다.

https://github.com/tistory-projects/tistory-skin-hELLO/blob/c01c72865e0056ed921d2ca644bc7caa38375829/views/Main/Paging.pug#L15-L38

페이징의 번호의 디스플레이하곤 별개로 화살표도 네비게이션이 되어야 하는데 안 되는 거는 버그가 맞군요.

**Reply by Jaeppetto** [reply]
답변 감사드립니다 :D
